templates/results-row.php

```diff
@@ -14,5 +14,13 @@
 	<td>
 		{{data.message}}
 	</td>
+	<# if ( data.hasLinks ) { #>
+		<td>
+			<a href="{{data.link}}"<?php echo ( ! has_filter( 'wp_plugin_check_validation_error_source_file_editor_url_template' ) ) ? ' target="_blank"' : '' ?>>
```

Comment:
Why only make the link external when it's another editor, and not when it's the plugin file editor? I'm not sure about it, but am thinking opening in a new tab would be a better experience as otherwise you would lose where you were in the original plugin checker report.

My original feedback on accessibility was primarily about the screen reader text that we need to have on external links. Note that right now this is inconsistent either way, since the screen reader text should only be there when the link also opens an external window (i.e. has `target="_blank"`).

I would suggest to simply remove the condition here, as it may be unreliable anyway - different links may or may not use the filter, we don't know that just by checking whether there is a filter.
```suggestion
			<a href="{{data.link}}" target="_blank">
```